src/Sarif/Autogenerated/Rule.cs

```diff
@@ -80,6 +80,12 @@ public SarifNodeKind SarifNodeKind
         [DataMember(Name = "helpUri", IsRequired = false, EmitDefaultValue = false)]
         public Uri HelpUri { get; set; }
 
+        /// <summary>
+        /// Provides the primary documentation for the rule, useful when there is no on-line documentation.
```

Comment:
>on-line [](start = 85, length = 7)

ditto

Reply:
(Both of those comments come from the schema, so only one change is needed.)